### ISSUE TITLE
In OMEdit, generate Text(textColor=...) instead of Text(lineColor=...)

### ISSUE THREAD

**Issue by mwetter** [OpenModelica/OpenModelica#8245]
### Description

In OMEdit, texts are rendered as
```modelica
model test
equation

annotation(
    Diagram(graphics = {Text(origin = {-53, 31}, 
  lineColor = {204, 0, 0}, extent = {{-35, 37}, {35, -37}}, textString = "text")}));
end test;
```

### Steps to Reproduce

Generate the above model in OMEdit

### Expected Behavior

The `lineColor` annotation should be `textColor` as `lineColor` is deprecated.
The Modelica Language Specification says
> The contents inherited from FilledShape is deprecated, but kept for compatibility reasons
### Version and OS

 - OpenModelica Version: 1.19.0~dev-429-g962c211

**Comment by casella** [comment]
@adeas31 if this is an easy fix it would be nice to have it in 1.19.0

**Comment by casella** [comment]
Thanks!
